deploy/deploy_webhook.sh

```diff
@@ -0,0 +1,10 @@
+#! /bin/bash
+
+# Deploys the webhook binary if it is not currently up
+if ( curl -X POST http://127.0.0.1:10000/infra/your-deploy-key-here > /dev/null )
```

Comment:
You should let the deploy key be pulled in by the `environment.sh` declaration. This might confuse someone into uploading the secret key to GH

Reply:
Excellent catch.